### PR TITLE
fix description and journal writings for stock entry by a purchase

### LINKED ISSUE #2083: Missing description for stock entry in Journal
The stock entry operation for a purchase generate correctly writings in the Journal but the description is missing and it is impossible to post transaction into the general ledger.

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR fix : 
- The missing description in the journal when a stock is entered by purchase order #2083 
- Not necessary multiple writing in the journal for a stock entry

close #2083

### CODE REVIEW

File: server/controllers/stock/index.js [hunk 1]
```diff
@@ -92,23 +92,28 @@ function createStock(req, res, next) {
...
     // adding a lot insertion query into the transaction
     transaction.addQuery(createLotQuery, [createLotObject]);
 
     // adding a movement insertion query into the transaction
     transaction.addQuery(createMovementQuery, [createMovementObject]);
+  });
 
-    // An arry of common info, to send to the store procedure in order to insert to the posting journal
-    commonInfos = [ db.bid(document.uuid), document.date, req.session.enterprise.id, req.session.project.id, req.session.enterprise.currency_id, document.user ];
+  // An arry of common info, to send to the store procedure in order to insert to the posting journal
```
Comment (jniles) [inline server/controllers/stock/index.js:105]: :dancer:

**Review by jniles — APPROVED** [review]
This looks fine to me.  I'll wait on @DedrickEnc's opinion before merge.

**Review by DedrickEnc — APPROVED** [review]
LGTM

**Comment by jniles** [comment]
bors r=jniles,dedrickenc

**Pull request merged**